### PR TITLE
Related to cleaning strategy of Do.W.C.

### PR DESCRIPTION
**Pull request by damm89**
Related to: https://github.com/GemeenteUtrecht/dowc/pull/21.

### CODE REVIEW

File: backend/src/zac/contrib/dowc/api.py [hunk 1]
```diff
@@ -22,11 +22,14 @@ def get_client(user: User) -> Client:
...
     service = config.service
 
-    # override the actual logged in user in the `user_id` claim, so that D.O.C. is
+    # override the actual logged in user in the `user_id` claim, so that Do.W.C. is
     # aware of the actual end user
+    claims = {}
     if user is not None:
         service.user_id = user.username
-    client = service.build_client()
+        claims["user_email"] = user.email
```
Comment (sergei-maertens) [inline backend/src/zac/contrib/dowc/api.py:30]: See my other review here: https://github.com/maykinmedia/zgw-auth-backend/pull/1#pullrequestreview-610682741

File: backend/src/zac/contrib/dowc/views.py [hunk 2]
```diff
@@ -54,8 +54,9 @@ def post(self, request, bronorganisatie, identificatie, purpose):
         Create a dowc object in the dowc API and expose the document through a URL.
         """
         document = self.get_object(bronorganisatie, identificatie)
-        dowc_response, status_code = get_doc_info(request.user, document, purpose)
-        serializer = self.serializer_class(dowc_response)
+        referer = request.META.get("HTTP_REFERER", "")
```
Comment (sergei-maertens) [inline backend/src/zac/contrib/dowc/views.py:57]: PSA: in modern Django versions you can use [`request.headers`](https://docs.djangoproject.com/en/2.2/ref/request-response/#django.http.HttpRequest.headers)

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

File: backend/src/zac/contrib/dowc/tests/test_dowc_api.py [hunk 3]
```diff
@@ -122,7 +122,7 @@ def test_client(self, m):
         token = header.split(" ")[1]
         claims = jwt.decode(token, verify=False)
         self.assertEqual(claims["user_id"], self.user.username)
-
+        self.assertEqual(claims["zds"], {"email": self.user.email})
```
Comment (sergei-maertens) [inline backend/src/zac/contrib/dowc/tests/test_dowc_api.py:125]:
```suggestion
        self.assertEqual(claims["email"], self.user.email)
```
and of course, update to gemma-zds-client 1.0 :-)

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

File: backend/src/zac/contrib/dowc/tests/test_dowc_api.py (hunk 3 above)
Comment (damm89) [reply backend/src/zac/contrib/dowc/tests/test_dowc_api.py:125]: :partying_face:

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**